sale_prodlot_mrp/stock.py

```diff
@@ -0,0 +1,72 @@
+# -*- coding: utf-8 -*-
+##############################################################################
+#
+#  licence AGPL version 3 or later
+#  see licence in __openerp__.py or http://www.gnu.org/licenses/agpl-3.0.txt
+#  Copyright (C) 2014 Akretion (http://www.akretion.com).
+#  @author David BEAL <david.beal@akretion.com>
+#
+##############################################################################
+
+from openerp.osv import orm
+
+
+class StockMove(orm.Model):
```

Comment:
In fact we should explode this module in 2 modules.
The first module is sale_order_lot_mrp (that depend on sale_order_lot_selection and mrp)
This module add the lot into the production order and also pass the production.id into the context

The second module should be sale_order_lot_generator_mrp (that depend on sale_order_lot_mrp and sale_order_lot_generator). This module will generate automatically a prodlot for the component of the BOM with the option "phantom" that have the option "track_from_sale". An real exemple is when you sold a customisable product that will be delivered in two part. Then your product have a phamton bom and the option "track_from_sale" and when we explode it we have the two part of the production with an automatic prodlot number generated